Octokit/Models/Request/ApiOptions.cs

```diff
@@ -5,7 +5,7 @@
 namespace Octokit
 {
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
-    public class ApiOptions
+    public class ApiOptions 
```

Comment:
For some reasons VS do not show **ending** whitespaces in own diff tool. Is my mistake, fixed.